packages/client/src/forms/correction/payment.ts

```diff
@@ -46,8 +46,8 @@ export const correctionFeesPayment: IFormSectionGroup = {
             type: 'NUMBER',
             label: {
               defaultMessage: '',
-              id: 'form.field.label.totalFees',
-              description: 'Label for input Reason for Change'
+              id: 'form.field.label.empty',
```

Comment:
We probably shouldn't use the same id in many places if someone wants to one day make this text different

Reply:
@rikukissa tbh I think we need another issue to fix all the places that have an empty content key because it is required by types.